ci-config/validators.txt

```diff
@@ -0,0 +1,5 @@
+[validator_list_sites]
```

Comment:
Add a comment saying this file is required by the image but does nothing.

Reply:
Are you sure this file is needed for the container?

Reply:
I am. https://github.com/WietseWind/docker-rippled/blob/master/entrypoint#L18 this requires both files to exist to copy either of them.  Having this file empty though will throw errors in rippled that stanzas are missing.

Reply:
I've run rippled in standalone mode many times without any validators.txt file, which suggests that this is an issue specific to that Docker container and should probably be modified.

Reply:
Yes it is specific to that docker images entrypoint.

Reply:
Create WietseWind/docker-rippled/issues/18

Reply:
The logic referred to only runs if there's something in the `/config` folder in the container, if you map straight to the rippled config file path it'll work just fine (the entrypoint only executes that logic if there's a rippled config in the `/config` volume mapping in the first place.

Also: it doesn't error if not provided, it just doesn't copy. Meaning you can run with default config or map straight to the `/etc/opt/ripple/rippled.cfg` file in the container.

But: I separated the logic for the two files & pushed a new container image.

Reply:
@WietseWind thank you.  I am still  relatively new to docker and didn't think to map directly.